web/frontend/package.json

```diff
@@ -22,11 +22,16 @@
   "dependencies": {
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
+    "axios": "^0.24.0",
+    "formik": "^2.2.9",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-redux": "^7.2.5",
     "react-router-dom": "^5.3.0",
     "redux": "^4.1.1",
-    "redux-devtools-extension": "^2.13.9"
+    "redux-devtools-extension": "^2.13.9",
+    "redux-thunk": "^2.4.0",
```

Comment:
Why we need this package?

Reply:
I thought to use async actions when submitting the form, but when I wrote them, they worked exactly the same as sync actions.
Forgot to remove from package.json